packages/synapse-react-client/src/components/ChallengeDataDownload/ChallengeDataTable.tsx

```diff
@@ -84,12 +84,12 @@ export const ChallengeDataTable: React.FunctionComponent<DetailsViewProps> = ({
     // It's likely that the user will be throttled by the Synapse backend and may be waiting a
     // noticeable amount of time for the current request, so cancel it (in addition to cancelling future requests)
     if (getChildrenInfiniteRequestObject) {
-      queryClient.cancelQueries(
-        keyFactory.getEntityChildrenQueryKey(
+      queryClient.cancelQueries({
+        queryKey: keyFactory.getEntityChildrenQueryKey(
           getChildrenInfiniteRequestObject,
           true,
         ),
-      )
+      })
```

Comment:
`cancelQueries` and `invalidateQueries` now requires an argument that is an object, where the QueryKey is included in the `queryKey` property.